resources/js/components/RoomTypeChangeButton.vue

```diff
@@ -101,6 +118,11 @@ function handleOk() {
   }
 }
 
+function resetRoomTypeSelectEmits() {
+  roomTypeSelectBusy.value = false;
+  roomTypeSelectLoadingError.value = false;
+}
+
```

Comment:
What are the pros/cons for adding this to the `editRoomType` function

Reply:
The main difference is that the current solution resets the values when closing the dialog and that a solution with adding this to the `editRoomType` function would reset the values when opening the dialog. 

Pros for adding this to the `editRoomType` function:
* no new function just for resetting when we already have a function where the dialog data is reset
* resetting is handled in a way that is similar to how we normally handle resetting the dialogs and `@hide` inside the dialog would not be needed anymore _(only other place where `@hide` is used is inside the RoomCreateComponent)_

Cons:
- Reset is currently handled similar to the RoomCreateComponent which also contains the RoomTypeSelect and therefore `roomTypeSelectBusy` and `roomTypeSelectLoadingError`
- Current function name is a little clearer (clearer that values are reset)


I think adding this to the `editRoomType` function would be the better solution but I think that we then should perhaps also think about modifying the RoomCreateComponent to something similar

Reply:
Agree, the handling should be consistent